minet/cli/youtube/videos.py

```diff
@@ -59,6 +68,14 @@ def get_data(data_json):
         if not like_count:
             no_stat_likes = '1'
 
+        if caption == 'true':
```

Comment:
`caption` n'est pas un booléen (c'est possible)?

Reply:
@paubre what of it?

Reply:
oui ce n'est pas un booléen :/